lib/documentLoader.js

```diff
@@ -3,13 +3,39 @@
  */
 'use strict';
 
+/*eslint max-len: ["error", { "ignoreComments": true }]*/
+
 // load locally embedded contexts
 const contexts = require('./contexts');
 
+/**
+ * A utility module that provides a set of functions for using or
+ * extending jsonld-signature's built-in JSON-LD document loader.
+ * @module documentLoader
```

Comment:
Can we just remove this module tag? This is an internal file, we don't need to expose docs on the module itself as if it's a publicly available thing. Only the methods matter.

```suggestion
```